src/spotify-pupeteer/index.js

```diff
@@ -116,8 +116,8 @@ async function postEpisode(youtubeVideoInfo) {
   }
 
   async function loginAndWaitForNewEpisodeWizard() {
-    if (env.ANCHOR_LOGIN) {
-      await anchorLogin();
+    if (env.SPOTIFY_LOGIN) {
+      await spotifyLogin();
     } else {
```

Comment:
I believe there are two functions now with the same name. Could you check that this change?

Reply:
I think we won't need any if statements here since the only login type will be spotofy.